velox/dwio/parquet/tests/reader/ParquetReaderBenchmark.cpp

```diff
@@ -399,675 +408,1503 @@ int main(int argc, char** argv) {
-run(List_Filter_100_Nulls_20_next_50k_dict)                509.59ms     1.96
-run(List_Filter_100_Nulls_20_next_50k_plain)               487.91ms     2.05
-run(List_Filter_100_Nulls_20_next_100k_dict)               551.87ms     1.81
-run(List_Filter_100_Nulls_20_next_100k_plain)              444.48ms     2.25
-----------------------------------------------------------------------------
-run(List_Filter_100_Nulls_50_next_5k_dict)                 398.44ms     2.51
-run(List_Filter_100_Nulls_50_next_5k_plain)                351.05ms     2.85
-run(List_Filter_100_Nulls_50_next_10k_dict)                358.92ms     2.79
-run(List_Filter_100_Nulls_50_next_10k_Plain)               361.92ms     2.76
-run(List_Filter_100_Nulls_50_next_20k_dict)                365.71ms     2.73
-run(List_Filter_100_Nulls_50_next_20k_plain)               379.67ms     2.63
-run(List_Filter_100_Nulls_50_next_50k_dict)                386.53ms     2.59
-run(List_Filter_100_Nulls_50_next_50k_plain)               333.08ms     3.00
-run(List_Filter_100_Nulls_50_next_100k_dict)               391.00ms     2.56
-run(List_Filter_100_Nulls_50_next_100k_plain)              314.15ms     3.18
-----------------------------------------------------------------------------
-run(List_Filter_100_Nulls_70_next_5k_dict)                 269.02ms     3.72
-run(List_Filter_100_Nulls_70_next_5k_plain)                237.83ms     4.20
-run(List_Filter_100_Nulls_70_next_10k_dict)                240.33ms     4.16
-run(List_Filter_100_Nulls_70_next_10k_Plain)               242.11ms     4.13
-run(List_Filter_100_Nulls_70_next_20k_dict)                243.19ms     4.11
-run(List_Filter_100_Nulls_70_next_20k_plain)               262.23ms     3.81
-run(List_Filter_100_Nulls_70_next_50k_dict)                241.58ms     4.14
-run(List_Filter_100_Nulls_70_next_50k_plain)               237.81ms     4.21
-run(List_Filter_100_Nulls_70_next_100k_dict)               262.14ms     3.81
-run(List_Filter_100_Nulls_70_next_100k_plain)              238.90ms     4.19
-----------------------------------------------------------------------------
-run(List_Filter_100_Nulls_100_next_5k_dict)                 49.10ms    20.37
-run(List_Filter_100_Nulls_100_next_5k_plain)                49.41ms    20.24
-run(List_Filter_100_Nulls_100_next_10k_dict)                49.93ms    20.03
-run(List_Filter_100_Nulls_100_next_10k_Plain)               49.90ms    20.04
-run(List_Filter_100_Nulls_100_next_20k_dict)                49.38ms    20.25
-run(List_Filter_100_Nulls_100_next_20k_plain)               48.86ms    20.47
-run(List_Filter_100_Nulls_100_next_50k_dict)                48.91ms    20.45
-run(List_Filter_100_Nulls_100_next_50k_plain)               49.07ms    20.38
-run(List_Filter_100_Nulls_100_next_100k_dict)               49.25ms    20.31
-run(List_Filter_100_Nulls_100_next_100k_plain)              48.90ms    20.45
+run(ShortDecimalType_Filter_0_Nulls_0_next_5k_d            21.72ms     46.04
```

Comment:
Actually, let's not remove the existing Intel results. Just add the the end of your AMD test results